lectures/markov_chains_II.md

```diff
@@ -381,21 +343,11 @@ $$
 
 The diagram of the Markov chain shows that it is **irreducible**
 
-```{code-cell} ipython3
-:tags: [hide-input]
-
-dot = Digraph(comment='Graph')
-dot.attr(rankdir='LR')
-dot.node("0")
-dot.node("1")
-
-dot.edge("0", "1", label="1.0", color='red')
-dot.edge("1", "0", label="1.0", color='red')
-
-dot
+```{image} /_static/lecture_specific/markov_chains_II/example4.png
+:name: mc_example4
+:align: center
 ```
 
-
 In fact it has a periodic cycle --- the state cycles between the two states in a regular way.
 
 This is called [periodicity](https://www.randomservices.org/random/markov/Periodicity.html).
```

Comment:
I usually link to wikipedia, since the links there are pretty stable.

Reply:
Many thanks @jstac. The reseason why I use this link is because there is no good entry for periodicity on Wikipedia. Please see: https://en.wikipedia.org/wiki/Periodicity. Would you say [this link](https://stats.libretexts.org/Bookshelves/Probability_Theory/Probability_Mathematical_Statistics_and_Stochastic_Processes_(Siegrist)/16%3A_Markov_Processes/16.05%3A_Periodicity_of_Discrete-Time_Chains#:~:text=A%20state%20in%20a%20discrete,limiting%20behavior%20of%20the%20chain.) from libretexts is a better one, as it is also very stable.

Reply:
Hi @jstac,

Too many things are happening, so you may have missed this : ). If the link is fine, we can get this merged once I update the link. 

Many thanks in advance.

Reply:
Thanks for pinging me on this @HumphreyYang .  You are right, the libretexts one is good.  Let's go with that.